File: CodeWars/Python/8Kyu/CountTheMonkeys.py
def monkey_count(n):

    monkeys = [1]
    
    if n == 1:
        return monkeys
    else: 
        count = 1 
        while len(monkeys) < n:
            print(count)
            count+=1
            monkeys.append(count)

        return monkeys

print(monkey_count(10))

File: CodeWars/Java/8Kyu/CountTheMonkeys.java
public class CountTheMonkeys {
	public static int[] monkeyCount(final int n){
		int[]count = new int[n];
		for(int i = 0; i < n; i++){
			count[i] = i+1;
			System.out.println(count[i]);
		}
		return count;
		}
	
	public static void main(String[] args) {
			int n = 15;
			System.out.println(monkeyCount(n));
		}
}